api/v1alpha1/queuedworkload_types.go

```diff
@@ -39,6 +39,13 @@ type QueuedWorkloadSpec struct {
 
 	// admission holds the parameters of the admission of the workload by a ClusterQueue.
 	Admission *Admission `json:"admission,omitempty"`
+
+	// priority determines the order of access to the resources in the
+	// ClusterQueue associated with the queue where the workload is located.
```

Comment:
instead of located, should we say "published" or perhaps "posted"?

It makes me think that we need a glossary of terms for our user documentation.

Reply:
`associated` can also be a candidate?